client/actions/appointmentAction.js

```diff
@@ -33,9 +33,7 @@ export function getAppointment(parameter) {
             .then((response) => {
                 dispatch(getAppointmentAction(response.data))
             })
-            .catch((err) => {
-                console.log(err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}